packages/commonwealth/client/scripts/views/pages/overview/index.tsx

```diff
@@ -18,8 +16,16 @@ import { TopicSummaryRow } from './TopicSummaryRow';
 
 const OverviewPage = () => {
   const navigate = useCommonNavigate();
-  const { isWindowSmallInclusive } = useBrowserWindow({});
-  const user = useUserStore();
+  const topicNameFromURL = splitAndDecodeURL(location.pathname);
+  useEffect(() => {
+    if (topicNameFromURL === 'overview') {
+      const params = new URLSearchParams();
+      params.set('tab', 'overview');
```

Comment:
you'd have to update `HeaderWithFilters.tsx -> onFilterSelect()` logic for URL params setting to work correctly with this section. ex: `?featured=oldest or anything after '?'` should get removed when switching from 'all' to 'overview' tab.